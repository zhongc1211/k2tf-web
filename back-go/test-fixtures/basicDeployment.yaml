apiVersion: apps/v1
kind: Deployment
metadata:
  name: baz-app
  namespace: bat
  annotations:
    foo: fam
spec:
  replicas: 2
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      annotations:
        foo: fam
      labels:
        app: nginx
    spec:
      containers:
        - name: nginx
          image: nginx
          args: ["--debug", "--test"]
          ports:
            - port: 80
              containerPort: 80
          resources:
            requests:
              cpu: "1"
            limits:
              memory: "1Gi"
