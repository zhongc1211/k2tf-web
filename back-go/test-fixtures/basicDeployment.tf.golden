resource "kubernetes_deployment" "baz_app" {
  metadata {
    name        = "baz-app"
    namespace   = "bat"
    annotations = { foo = "fam" }
  }
  spec {
    replicas = 2
    selector {
      match_labels = { app = "nginx" }
    }
    template {
      metadata {
        labels      = { app = "nginx" }
        annotations = { foo = "fam" }
      }
      spec {
        container {
          name  = "nginx"
          image = "nginx"
          args  = ["--debug", "--test"]
          port {
            container_port = 80
          }
          resources {
            limits   = { memory = "1Gi" }
            requests = { cpu = "1" }
          }
        }
      }
    }
  }
}
